src/euler_types.jl

```diff
@@ -0,0 +1,220 @@
+# File created: 08-Mar-2016 15:00:09
+# using Matlab 2013b with Symbolic Math Toolbox version 5.11
+
+
+##############################
+# Euler Representations
+##############################
+
+"""
+EulerOrder specifies the extrinsic (axes of rotation are fixed) order of rotations
+"""
+abstract EulerOrder     # Defines the order of rotations
+
+
+"""
+EulerOrder specifies the extrinsic (axes of rotation are fixed) order of rotations
```

Comment:
Fixed

On 10 May 2016 at 13:01, Chris Foster <notifications@github.com> wrote:

> In src/euler_types.jl
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62606406>:
>
> > +# File created: 08-Mar-2016 15:00:09
> > +# using Matlab 2013b with Symbolic Math Toolbox version 5.11
> > +
> > +
> > +##############################
> > +# Euler Representations
> > +##############################
> > +
> > +"""
> > +EulerOrder specifies the extrinsic (axes of rotation are fixed) order of rotations
> > +"""
> > +abstract EulerOrder     # Defines the order of rotations
> > +
> > +
> > +"""
> > +EulerOrder specifies the extrinsic (axes of rotation are fixed) order of rotations
>
> Broken comment?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62606406>
>